.github/workflows/update-helm-repo.yaml

```diff
@@ -55,7 +55,9 @@ jobs:
         with:
           fetch-depth: 0
           path: source
-          persist-credentials: false
+          # zizmor may complain about this as a "medium" level warning, but it is necessary
+          # to be able to push the release tags later on in this workflow
+          persist-credentials: true
```

Comment:
fwiw security team [okay'd](https://raintank-corp.slack.com/archives/C08Q3NB2VL3/p1746203330203219?thread_ts=1746043218.511839&cid=C08Q3NB2VL3) this